### PR TITLE
Add ignore frames for Fuchsia.

### PR DESCRIPTION
**Pull request by oliverchang**

### CODE REVIEW

File: src/python/crash_analysis/stack_parsing/stack_analyzer.py [hunk 1]
```diff
@@ -511,8 +511,12 @@
     r'^panic$',
     r'^runtime\.',
 
-    # Fuchsia libFuzzer specific.
+    # Fuchsia specific.
     r'^CrashTrampolineAsm',
+    r'^libc_io_functions_not_implemented_use_fdio_instead',
+    r'^\<libclang_rt.asan.so\>',
+    r'^__zx_panic',
```
Comment (inferno-chromium) [inline src/python/crash_analysis/stack_parsing/stack_analyzer.py:518]: nit: for python 2, _ needs to be escaped in all these entries.

Comment (oliverchang) [reply src/python/crash_analysis/stack_parsing/stack_analyzer.py]: Done!

**Pull request merged**